README.md

```diff
@@ -322,4 +322,4 @@ In markdown, images should be referenced by their absolute path, starting with `
 
 ## Redirects
 
-For now, redirects will need to be handled here: https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js
\ No newline at end of file
+For now, redirects will need to be handled here: https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js
```

Comment:
```suggestion
Handle all redirects in this file in the `terraform-website` repository: https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js
```

Suggest changing "For now, redirects will need to be handled here:" to "Handle all redirects in this file in the `terraform-website` repository:" to avoid future tense and passive voice style issues.

Reply:
Fine with me!
(I did not change that line, it was just marked as such because the Github UI seemed to have removed or added a newline at the end of the file 😄)